src/cli.js

```diff
@@ -475,8 +491,11 @@ function cli (inputArgs) {
             link: args.link || false,
             save: args.save,
             shrinkwrap: args.shrinkwrap || false,
-            force: args.force || false
+            force: args.force || false,
+            production: args.production,
+            noprod: args.noprod || false
```

Comment:
no need to pass noprod, it is only used to figure out production. Which you do on L454